### PR TITLE
fix: fix non-olm sso test and use set for env variable

### PR DESCRIPTION
**Pull request by saumeya**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

> /kind bug

Depends on merge of this PR - https://github.com/redhat-developer/gitops-operator/pull/594
**What does this PR do / why we need it**:

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**Test acceptance criteria**:

* [ ] Unit Test
* [ ] E2E Test

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: test/openshift/e2e/sequential/1-050_validate_sso/04-disable-dex-through-sub.yaml
Comment (svghadi) [inline test/openshift/e2e/sequential/1-050_validate_sso/04-disable-dex-through-sub.yaml:1]: We have dropped the support for `DISABLE_DEX` env in latest argocd-operator. This change can be skipped. 

dex or keycloak can be disabled by removeing `.spec.sso` field from ArgoCD CR.

File: test/openshift/e2e/sequential/1-050_validate_sso/05-unset-disable-dex-env-var.yaml
Comment (svghadi) [inline test/openshift/e2e/sequential/1-050_validate_sso/05-unset-disable-dex-env-var.yaml:1]: Same as previous comment.

**Review by svghadi — CHANGES_REQUESTED** [review]

File: test/openshift/e2e/sequential/1-050_validate_sso/04-disable-dex-through-sub.yaml
Comment (saumeya) [reply test/openshift/e2e/sequential/1-050_validate_sso/04-disable-dex-through-sub.yaml:1]: Will remove this test, thanks

**Review by anandf — APPROVED** [review]

**Review by svghadi — APPROVED** [review]

**Pull request merged**